.golangci.yml

```diff
@@ -13,11 +13,8 @@ issues:
   exclude-use-default: false
 
 linters-settings:
-  gofmt:
-    simplify: false
```

Comment:
[`true` is a default value](https://golangci-lint.run/usage/linters/#gofmt) for golangci